src/test/java/io/scif/writing/TiffFormatTest.java

```diff
@@ -210,4 +211,26 @@ public void testWriteToNewLocation() throws IOException {
 		opener.openImgs(saveLocation);
 	}
 
+	/**
+	 * Verify we get an {@link ImgIOException} when writing to a destination that
+	 * already exists.
+	 */
+	@Test(expected = io.scif.img.ImgIOException.class)
+	public void testFailIfOverwriting() throws IOException {
+		testOverwritingBehavior();
+	}
+
+	/**
+	 * Verify we can overwrite an existing file with {@link SCIFIOConfig}.
+	 * <p>
+	 * NB: for TIFF this technically appends the image
```

Comment:
In future, it might be good to actually test that it appended, rather than just writing a valid TIFF. But good enough for now! :+1: